designs/wahid/src/front.js

```diff
@@ -273,22 +259,41 @@ export default (part) => {
         90
       )
     }
-
     // Facing/Lining boundary (flb)
     paths.flbFacing = new Path()
       .move(points.dartTop)
       .curve(points.flbCp, points.flbCpTop, points.flbTop)
       .attr('class', 'fabric')
     paths.flbLining = paths.flbFacing.clone().attr('class', 'lining dashed')
 
+    // Grainline
+    let grainlineVariableLength = points.closureTop.dist(points.cfHips)
```

Comment:
There's a lot going on here. Any reason this won't do?

```js
macro('grainline', {
  to: points.neck,
  from: new Point(points.neck.x, points.cfHem.y)
})
```

Reply:
Nope, that would work just fine (: